cimi-resources/src/com/sixsq/slipstream/ssclj/resources/connector_template.clj

```diff
@@ -159,6 +166,15 @@
     :readOnly    false
     :order       10}
 
+   :objectStoreEndpoint
+   {:displayName "object.store.endpoint"
+    :type        "string"
+    :category    ""
+    :description "Cloud Object Store Service endpoint (e.g. http://s3.example.com:5000)"
+    :mandatory   true
```

Comment:
Is the `:objectStoreEndpoint` key mandatory?  It is marked as mandatory here and is also mandatory in the "dummy" connector.  In contrast, it is optional in the "alpha" example connector template.